pkg/webhook/validation/dynakube/oneagent_test.go

```diff
@@ -424,3 +424,41 @@ func createDynakubeWithHostGroup(args []string, hostGroup string) *dynatracev1be
 		},
 	}
 }
+
+func TestValidateOneAgentVersionIsSemVer(t *testing.T) {
+	testCasesAcceptedVersions := []string{"", "1.0.0", "1.200.1"}
+
+	testCasesNotAcceptedVersions := []string{"latest", "raw", "1.200.1-raw", "1.200.1+build", "1.200", "1", "1.0", "v1.200.0"}
+
+	for _, tc := range testCasesAcceptedVersions {
+		t.Run("should accept version "+tc, func(t *testing.T) {
+			assertAllowedResponseWithoutWarnings(t, &dynatracev1beta2.DynaKube{
+				ObjectMeta: defaultDynakubeObjectMeta,
+				Spec: dynatracev1beta2.DynaKubeSpec{
+					APIURL: testApiUrl,
+					OneAgent: dynatracev1beta2.OneAgentSpec{
+						ClassicFullStack: &dynatracev1beta2.HostInjectSpec{
+							Version: tc,
+						},
+					},
+				},
+			})
+		})
+	}
+
+	for _, tc := range testCasesNotAcceptedVersions {
+		t.Run("should accept version "+tc, func(t *testing.T) {
+			assertDeniedResponse(t, []string{"Only semantic versions in the form of major.minor.patch (e.g. 1.0.0) are allowed!"}, &dynatracev1beta2.DynaKube{
+				ObjectMeta: defaultDynakubeObjectMeta,
+				Spec: dynatracev1beta2.DynaKubeSpec{
+					APIURL: testApiUrl,
```

Comment:
since you touched parts of the file, would it make senes to change the imports in this file aswell? so its dynakube instead of dynatracev1beta2?